wasm.md

```diff
@@ -276,38 +280,44 @@ Comparisons consume two operands and produce a bool, which is an `i32` value.
 
 ```k
     syntax PlainInstr ::= IValType "." IRelOp
- //                     | FValType "." FRelOp
+                        | FValType "." FRelOp
  // -----------------------------------------
 
     syntax Instr ::= IValType "." IRelOp Int   Int
- //                | FValType "." FRelOp Float Float
+                   | FValType "." FRelOp Float Float
  // ------------------------------------------------
     rule <k> ITYPE . ROP:IRelOp => ITYPE . ROP C1 C2 ... </k>
-         <valstack> < ITYPE > C2 : < ITYPE > C1 : VALSTACK => VALSTACK  </valstack>
+         <valstack> < ITYPE > C2 : < ITYPE > C1 : VALSTACK => VALSTACK </valstack>
+    rule <k> FTYPE . ROP:FRelOp => FTYPE . ROP C1 C2 ... </k>
+         <valstack> < FTYPE > C2 : < FTYPE > C1 : VALSTACK => VALSTACK </valstack>
+
 ```
 
 ### Conversion Operations
 
 Conversion operators always take a single argument as input and cast it to another type.
-For each element added to `ConvOp`, functions `#convSourceType` and `#convOp` must be defined over it.
+For each element added to `CvtOp`, functions `#convSourceType` and `#CvtOp` must be defined over it.
 
 These operators convert constant elements at the top of the stack to another type.
 The target type is before the `.`, and the source type is after the `_`.
 
 ```k
-    syntax PlainInstr ::= IValType "." ConvOp
- //                     | FValType "." ConvOp
- // -----------------------------------------
+    syntax PlainInstr ::= IValType "." CvtOp
+                        | FValType "." CvtOp
+ // ----------------------------------------
 
-    syntax Instr ::= IValType "." ConvOp Int
- //                | FValType "." ConvOp Float
+    syntax Instr ::= IValType "." CvtOp Number
+                   | FValType "." CvtOp Number
  // ------------------------------------------
-    rule <k> ITYPE . CONVOP:ConvOp => ITYPE . CONVOP C1  ... </k>
+    rule <k> ITYPE:IValType . CVTOP:CvtOp => ITYPE . CVTOP C1  ... </k>
```

Comment:
Why do we need two separate rules here? They look identical to me, except the typing of the `ValType`. But both `IValType` and `FValType` as subsorts of `ValType`, right?

Reply:
fixed